packages/lib/delegationCredential/server.ts

```diff
@@ -104,7 +110,18 @@ const _buildDelegatedCalendarCredential = ({
   delegationCredential: DelegationCredential;
   user: User;
 }) => {
-  log.debug("buildDelegationCredential", safeStringify({ delegationCredential, user }));
```

Comment:
Entire delegationCredential can have serviceAccount which we don't want to log in debug mode too.